real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/RobobuggyGUITabs.java

```diff
@@ -0,0 +1,32 @@
+package com.roboclub.robobuggy.ui;
+
+import javax.swing.JComponent;
+import javax.swing.JTabbedPane;
+
+/**
+ * 
+ * @author Trevor Decker
+ * 
+ * is a robobuggy gui element that contains many other gui elements in seperate tabs
+ *
+ */
+public class RobobuggyGUITabs extends JTabbedPane {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Each class should declare at least one constructor](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1939509492/issues/source?bid=2938665&fileBranchId=3104187#l13)